nodeup/pkg/model/kops_controller.go

```diff
@@ -87,6 +88,19 @@ func (b *KopsControllerBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 	if model.UseCiliumEtcd(b.Cluster) {
 		caList = append(caList, "etcd-clients-ca-cilium")
 	}
+	if model.UseJWTForBootstrap(b.Cluster) {
```

Comment:
Please don't add new references to the deprecated `b.Cluster`. This should be using something that tests `b.BootConfig.CloudProvider`.